specification/synapse/data-plane/Microsoft.Synapse/preview/2020-01-01-preview/bigDataPools.json

```diff
@@ -0,0 +1,189 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-01-01-preview",
+    "title": "SynapseClient"
```

Comment:
>SynapseClient [](start = 14, length = 13)

We got some comments from SDK team before. Let's name this as `BigDataPoolClient`

Reply:
Thanks, resolved it.